.github/workflows/on_prerelease.yaml

```diff
@@ -9,13 +9,14 @@ on:
 
 jobs:
   pre-release:
-    uses: newrelic/coreint-automation/.github/workflows/reusable_pre_release.yaml@v3
+    uses: newrelic/coreint-automation/.github/workflows/reusable_pre_release.yaml@v4
     with:
       tag: ${{ github.event.release.tag_name }}
       integration: "docker"
       test_package: false
       run_test_windows: false
       run_build-win-packages: false
       publish_schema: custom
+      dest_prefix: "infrastructure_agent/"
```

Comment:
You can leave it out and will take the default, isn't it?

Reply:
Yeah, I can leave it out. But left it as a documentation for what to change for testing. Probably I should add a comment. Let me add it.